recipe/meta.yaml

```diff
@@ -30,27 +26,35 @@ requirements:
     - libgcrypt
     - libgpg-error
     - libiconv  # [not linux]
-    - libtool
-    - libxml2
-    - libxslt
+    - libtool    
+    - libxml2 2.10
+    - libxslt 1.1.37
     - openssl
     - xz
```

Comment:
that is C/C++ shouldn't we have explicit pinning ?

Reply:
I would wait for the consensus. For me, it's still not persuasively enough because it will makes our work nightmarish

Reply:
Until this is formally adopted in our recipe standards, we shouldn't enforce this rule. We had an initial discussion, but not a formal decision.

Reply:
Why would it make our work nightmarish? Just trying to understand...